### DISCUSSION TITLE
pnpm not found

### DISCUSSION THREAD

**Discussion by zockinger** [ether/etherpad-lite#6248] [Q&A]
Trying to use the windows package V2.01. I think some components are missing:
d:\STUFF\Webserver\Etherpad.new>start.bat
'pnpm' is not recognized as an internal or external command,
operable program or batch file.

**Comment by SamTV12345** [comment]
Yes. Pnpm is the external program. Try npm install -g pnpm

**Reply by zockinger** [reply]
Sure, thats clear but the nice thing with V1.X was that you could extract the ZIP file and you didn't need to install anything else.
npm is not part of a standard windows installation, so thats another thing I'd need to install first.

**Reply by SamTV12345** [reply]
I mean if you install node.exe you should automatically have npm installed.

**Reply by zockinger** [reply]
I know, my point is that V1 is truly portable software, extract zip and go. 
It would be nice to have that in V2 as well.

**Reply by Gared** [reply]
@SamTV12345 Can we install pnpm during installOnWindows.bat locally? Like "npm install pnpm"?

**Comment by neorazor** [comment]
Anyone have a solution for this? Note this is a Windows installation issue... 

*EDIT* I've installed pnpm from pnpm.io and the error remains.

**Reply by SamTV12345** [reply]
Yes. You just need to do npm install -g pnpm. Maybe we should add a hint if the command fails.

**Reply by neorazor** [reply]
So download the package and install via the CLI...

Yeah, I followed the instructions from here: 

https://pnpm.io/installation

Which has you run the PowerShell installation script. 

I would think a hint would be appropriate.

**Reply by SamTV12345** [reply]
Absolutely I'll add that tomorrow. Sorry for the inconvenience 😅
